### PR TITLE
Put require statements into variables

### PR DESCRIPTION
**Pull request by AamirAnwar**
- Testing how to resolve conflicts
- Rebase to this repo's master
- Let's discuss this 😆

### CODE REVIEW

**Comment by AamirAnwar** [comment]
@sid093 I created a PR against your repo 🤣

File: index.js [hunk 1]
```diff
@@ -13,9 +16,9 @@ const _SERVER_PORT = process.env.PORT || 3000;
 app.use(bodyParser.json());
 app.use(cors());
 // Add API routes
-app.use('/api/auth', require('./server/api/auth.js'));
```
Comment (AamirAnwar) [inline index.js:16]: I think this will really help in terms of code sanctity 🥇

Comment (sidxdev) [reply index.js:16]: Yeah probably better to have a variable then inline usage.

**Comment by sidxdev** [comment]
Just btw. This repo doesn;t deploy to heroku. 
The most recent codebase is in gitlab repo.

**Comment by AamirAnwar** [comment]
Got it. Will head on over to GitLab

**Pull request merged**